content/events/19-20/mini-lan.md

```diff
@@ -1,40 +1,11 @@
 ---
-title: '[Geannuleerd] Geeks, Attack: Bootcamp!'
-description: Zeus' bootcamp LAN-Party!
+title: 'Geeks, Attack: Quarantine edition!'
+description: Van LAN-party naar WAN-party
 created_at: 05-03-2020
 time: '27-03-2020 18:00'
 end: '28-03-2020 4:00'
-facebook: 'https://www.facebook.com/events/648201269331401'
-location: 'Therminal, Hoveniersberg 24 9000 Gent'
-locationlink: 'Therminal'
+location: 'Bij je thuis'
+locationlink: ''
```

Comment:
I think this line should be deleted, otherwise the map is shown, but no location is found (just a guess, haven't tested this)

Reply:
If this is deleted, the entire post breaks and no content is shown :smile:

Reply:
In that case keep it, but also https://github.com/ZeusWPI/zeus.ugent.be/issues/382